### PR TITLE
feat: Buttons Component

### PR DESCRIPTION
**Pull request by munjalSaksham71**

### CODE REVIEW

**Comment by Namrata-J** [comment]
Buttons look good and also code is clean!!you can merge it!!

File: docs/docs.css [hunk 1]
```diff
@@ -78,7 +78,7 @@ li {
     overflow:hidden;
 } 
 
-.badge_component {
+.component {
     display: flex;
     flex-wrap: wrap;
     gap: 30px;
```
Comment (Raghav888) [inline docs/docs.css:84]: Use rem

**Review by Raghav888 — APPROVED** [review]
Good to merge and in the next PR please change all values to rem.

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me.

**Comment by munjalSaksham71** [comment]
> Good to merge and in the next PR please change all values to rem.

Okay Cool! i will keep it in mind

File: utils/button.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.btn {
+    padding: 8px 16px;
+    border: none;
+    cursor: pointer;
+}
+
+.btn-primary {
+    background-color: var(--primary);
+    color: #ffffff;
```
Comment (t007rushi) [inline utils/button.css:9]: use variables instead direct colors

File: utils/button.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    border: 1px solid var(--primary);
+    color: var(--primary);
+}
+
+.floating_btn {
+    padding: 1rem;
+    border-radius: 50%;
+    border: none;
+    font-weight: bolder;
+    font-size: 20px;
```
Comment (t007rushi) [inline utils/button.css:81]: use rem above 10px

**Review by t007rushi — COMMENTED** [review]
do minor changes & you are good to go

**Comment by ShreyashMahajan** [comment]
wow UI looks great and clean and naming of classes are also very clean and understandable.

**Pull request merged**